test/common/stream_info/stream_info_impl_test.cc

```diff
@@ -38,7 +38,7 @@ class StreamInfoImplTest : public testing::Test {
 protected:
   void assertStreamInfoSize(StreamInfoImpl stream_info) {
     ASSERT_TRUE(sizeof(stream_info) == 800 || sizeof(stream_info) == 816 ||
-                sizeof(stream_info) == 840)
+                sizeof(stream_info) == 840 || sizeof(stream_info) == 920)
         << "If adding fields to StreamInfoImpl, please check to see if you "
            "need to add them to setFromForRecreateStream or setFrom! Current size "
```

Comment:
Did you change these?

Reply:
Yes, there is a new field in stream_info (upstream_timing_) so there is new size to consider.

Reply:
Sorry... what I meant was that this comment here says "please check to see if you need to add them to setFromForRecreateStream or setFrom!". I'm asking if this is something you need to do for this new field?

Reply:
It should not be. Looking at setFromForRecreateStream and setFrom they only need downstream items

Reply:
`setFromForRecreateStream` is for downstream items, but the `setFrom` is to copy every field in streamInfo which includes upstream items, for example [upstream_info](https://github.com/envoyproxy/envoy/blob/a7fab06b8ec2500a421251781cc61d774e3ace60/source/common/stream_info/stream_info_impl.h#L353).

But i think we don't need to add the new fields in the first place.

Reply:
Irrelevant now due to changes.